core/controllers/admin.py

```diff
@@ -963,6 +964,24 @@ def post(self):
             raise self.InvalidInputException('This app cannot send emails.')
 
 
+class MemoryCacheHandler(base.BaseHandler):
+    """Handler for the memory cache."""
```

Comment:
Admin handler for managing the memory cache.

Reply:
Done